manifests/params.pp

```diff
@@ -161,7 +161,7 @@
   $puppetca_cmd       = $puppet::params::puppetca_cmd
   $puppet_group       = 'puppet'
   $puppetdir          = $puppet::params::dir
-  $autosign_location  = "${puppetdir}/autosign.conf"
+  $autosign_location  = $puppet::params::autosign
```

Comment:
Is this parameter entirely unused now?  This might necessitate removing it and bumping the major version, as it'll be inoperable.